.github/workflows/docs.yml

```diff
@@ -26,18 +26,13 @@ jobs:
           poetry --version
       - name: Install dependencies and library
         run: poetry install --no-root
-      - name: Generate documentation sources
-        run: |
-          poetry run sphinx-apidoc -f -o ./source ../scaleway-core
-          poetry run sphinx-apidoc -f -o ./source ../scaleway
-          poetry run sphinx-apidoc -f -o ./source ../scaleway-async
       - name: Generate documentation
-        run: poetry run sphinx-build -b html ./ ./_build
+        run: poetry run make html
```

Comment:
please add a `-j auto` flag to sphinx-docs build because it is really slow otherwise